zh/backup-to-aws-s3.md

```diff
@@ -0,0 +1,516 @@
+        iam.amazonaws.com/role: arn:aws:iam::123456789012:role/user
+    spec:
+      backupType: full
+      br:
+        cluster: demo1
+        sendCredToTikv: false
+        clusterNamespace: test1
+        # enableTLSClient: false
+        # logLevel: info
+        # statusAddr: <status-addr>
+        # concurrency: 4
+        # rateLimit: 0
+        # timeAgo: <time>
+        # checksum: true
+      from:
+        host: <tidb-host-ip>
+        port: <tidb-port>
+        user: <tidb-user>
+        secretName: backup-demo1-tidb-secret
+      s3:
+        provider: aws
+        region: us-west-1
+        bucket: my-bucket
+        prefix: my-folder
+    ```
+
++ 创建 `Backup` CR，通过 `IAM` 绑定 `ServiceAccount` 授权的方式备份集群:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    kubectl apply -f backup-aws-s3.yaml
+    ```
+
+    `backup-aws-s3.yaml` 文件内容如下：
+
+    ```yaml
+    ---
+    apiVersion: pingcap.com/v1alpha1
+    kind: Backup
+    metadata:
+      name: demo1-backup-s3
+      namespace: test1
+    spec:
+      backupType: full
+      serviceAccount: tidb-backup-manager
+      br:
+        cluster: demo1
+        sendCredToTikv: false
+        clusterNamespace: test1
+        # enableTLSClient: false
+        # logLevel: info
+        # statusAddr: <status-addr>
+        # concurrency: 4
+        # rateLimit: 0
+        # timeAgo: <time>
+        # checksum: true
+      from:
+        host: <tidb-host-ip>
+        port: <tidb-port>
+        user: <tidb-user>
+        secretName: backup-demo1-tidb-secret
+      s3:
+        provider: aws
+        region: us-west-1
+        bucket: my-bucket
+        prefix: my-folder
+    ```
+
+以上三个示例分别使用三种授权模式将数据到处到 Amazon S3 存储上。Amazon S3 的 `region`、`acl`、`endpoint`、`storageClass` 配置项均可以省略。其余非 Amazon S3 的但是兼容 S3 的存储均可使用和 Amazon S3 类似的配置。可参考上面例子中 Ceph 的配置，省略不需要配置的字段。
+
+Amazon S3 支持以下几种 access-control list (ACL) 策略：
+
+* `private`
+* `public-read`
+* `public-read-write`
+* `authenticated-read`
+* `bucket-owner-read`
+* `bucket-owner-full-control`
+
+如果不设置 ACL 策略，则默认使用 `private` 策略。这几种访问控制策略的详细介绍参考 [AWS 官方文档](https://docs.aws.amazon.com/AmazonS3/latest/dev/acl-overview.html)。
+
+Amazon S3 支持以下几种 `storageClass` 类型：
+
+* `STANDARD`
+* `REDUCED_REDUNDANCY`
+* `STANDARD_IA`
+* `ONEZONE_IA`
+* `GLACIER`
+* `DEEP_ARCHIVE`
+
+如果不设置 `storageClass`，则默认使用 `STANDARD_IA`。这几种存储类型的详细介绍参考 [AWS 官方文档](https://docs.aws.amazon.com/AmazonS3/latest/dev/storage-class-intro.html)。
+
+创建好 `Backup` CR 后，可通过如下命令查看备份状态：
+
+{{< copyable "shell-regular" >}}
+
+ ```shell
+ kubectl get bk -n test1 -o wide
+ ```
+
+更多 `Backup` CR 字段的详细解释:
+
+* `.spec.metadata.namespace`：`Backup` CR 所在的 namespace。
+* `.spec.from.host`：待备份 TiDB 集群的访问地址。
+* `.spec.from.port`：待备份 TiDB 集群的访问端口。
+* `.spec.from.user`：待备份 TiDB 集群的访问用户。
+* `.spec.from.tidbSecretName`：待备份 TiDB 集群所需凭证的 secret。
+* `.spec.storageClassName`: 备份时所需的 persistent volume (PV) 类型。如果不指定该项，则默认使用 TiDB Operator 启动参数中 `default-backup-storage-class-name` 指定的值，该值默认为 `standard`。
```

Comment:
It's not necessary for BR and `default-backup-storage-class-name` has been removed from tidb-operator.